data/extra_defs.asl

```diff
@@ -121,13 +121,13 @@ bits(148) GPRS;
 _R[integer n] = bits(32) value
     assert n >= 0 && n <= 14;
     bits(37) idx = integerToSBV(n);
-    GPRS = uninterpFn("gpr_set", GPRS, truncate(idx, 4), value);
+    GPRS = uninterpFn("uf_gpr_set", GPRS, truncate(idx, 4), value);
```

Comment:
Likely we should make the override for "uninterpFn" add this to the underlying function name (in Translation.hs) rather than putting it in manually here. In particular since all the floating point operations use it.

Reply:
That would probably work if we made a separate function to handle the uninterpreted function creation for `write_mem` and `read_mem` (which I think don't get those prefixes).  The floating point functions use `uninterpFnN`, which I think is already doing something like that

Reply:
If there are special cases where we don't want it then yeah, it's probably easier to just do it manually. I don't think `uninterpFnN` is adding it though, but it's likely that this is getting caught by `toUFIfNeeded` for the floating point functions.